tox.ini

```diff
@@ -20,6 +20,8 @@ deps =
     flake8-docstrings
     flake8-isort
     pep8-naming
+    # https://github.com/lyz-code/yamlfix/issues/286
+    maison>=1.4.0,<1.4.3
```

Comment:
yamlfix issued an update, so we can remove this now.

Reply:
Removed in bd17bc3